packages/gatsby/cache-dir/production-app.js

```diff
@@ -163,7 +163,8 @@ apiRunnerAsync(`onClientEntry`).then(() => {
                     })
                   } else {
                     return createElement(ComponentRenderer, {
-                      location: { page: true, pathname: `/404.html` },
+                      page: true,
```

Comment:
Changed it into this way instead, since `ComponentRenderer` is accepting and accessing `page` props instead of getting it from `location` props.